src/main/python/measure_judged.py

```diff
@@ -0,0 +1,69 @@
+"""Script to measure the percentage of judged documents at various
```

Comment:
Can you add the standard boilerplate with Apache 2 headers etc.?

Reply:
Sure